mysql/mysql-lib.pl

```diff
@@ -1304,6 +1304,37 @@ sub mysql_size_input
 		    [ "M", "MB" ], [ "G", "GB" ] ]);
 }
 
+# get_table_index_stats(db)
+# Retrieves index stats for all tables in the given database
+sub get_table_index_stats
+{
+my ($db) = @_;
+my $table_list = join(", ", map { "'$_'" } &list_tables($db));
```

Comment:
There's a quoting function in mysql-lib.pl you can use instead of doing it yourself

Reply:
I don't think we even need this with formal params ..

Reply:
Can a table name contain ' ?

Reply:
Oh never mind I see you fixed this